.github/workflows/lint.yml

```diff
@@ -12,10 +12,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - name: Get toolchain version
+        id: toolchain_version
+        run: echo "::set-output name=toolchain_version::$(grep channel rust-toolchain.toml | cut -d\" -f2)"
       - uses: actions-rs/toolchain@v1
         with:
           components: rustfmt
-          toolchain: nightly-2022-03-23
+          toolchain: ${{ steps.toolchain_version.outputs.toolchain_version }}
           profile: minimal
           override: true
```

Comment:
we should be able to replace this whole thing by `run: rustup show`
See https://github.com/profianinc/drawbridge/blob/39ea7b34576ce5edafa7e81aa0333c9298c4bf90/.github/workflows/test.yml#L8-L9

`rustup` is already installed on `ubuntu` images by default, `rustup show` will parse the `rust-toolchain.toml`
`rust-toolchain.toml` would need to contain all the components and targets for it to work